Automattic/wp-calypso
# DevDocs: Render a Gutenberg block

This continues #26065 and renders all the Gutenberg blocks from the `@wordpress/block-primary` package in the Calypso's DevDocs.

![screen shot 2018-08-13 at 22 52 50](https://user-images.githubusercontent.com/1233880/44057859-a9ea0108-9f4c-11e8-83c3-6687be28cfac.png)

### Testing Instructions

**Note**: The `devdocs/gutenberg-blocks` feature flag has to be enabled.
- Install dependencies with `npm install`
- Start calypso in a development environment with `npm start`
- Navigate to `/devdocs/gutenberg-blocks`
- The following blocks appear:
    - [x] ~~`core/archives`~~
    - [x] `core/audio`
    - [x] ~~`core/block`~~
    - [x] `core/button`
    - [x] ~~`core/categories`~~
    - [x] `core/code`
    - [x] `core/columns`
    - [x] `core/cover-image`
    - [x] ~~`core/embed`~~
    - [x] `core/file`
    - [x] `core/gallery`
    - [x] `core/heading`
    - [x] `core/image`
    - [x] ~~`core/latest-comments`~~
    - [x] ~~`core/latest-posts`~~
    - [x] `core/list`
    - [x] ~~`core/more`~~
    - [x] ~~`core/nextpage`~~
    - [x] `core/paragraph`
    - [x] `core/preformatted`
    - [x] `core/pullquote`
    - [x] `core/quote`
    - [x] `core/separator`
    - [x] `core/shortcode`
    - [x] `core/spacer`
    - [x] `core/subhead`
    - [x] `core/table`
    - [x] ~~`core/test`~~
    - [x] `core/text-columns`
    - [x] `core/verse`
    - [x] `core/video`

### Issues
- There are blocks that are rendered by a PHP server: `core/archives`, `core/block`, `core/categories`, `core/latest-comments` and `core/latest-posts`
- There are blocks that renders markup (HTML comments, shortcodes) that only makes sense in a post: `core/more`, `core/nextpage`, `core/shortcode` and `core/embed`
- ~~Content injected to blocks is wrapped by a `<div>`: `core/list`, `core/table`, `core/cover-image`, `core/paragraph`, `core/text-columns` and `core/columns`.~~ Solved by rendering the blocks with `serialize` from `@wordpress/blocks` and `RawHTML` from `@wordpress/element`.
- `core/gallery`: captions are exceeding the width of the images. It looks like there is a missing `box-sizing: border-box;
- `core/heading`: DevDocs styles overrides the heading elements styles
